Tests/Utils/HardcodedImageProvider.swift

```diff
@@ -0,0 +1,21 @@
+// Created by Cal Stephens on 1/19/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Lottie
+import QuartzCore
+#if os(iOS)
+import UIKit
+#endif
+
+// MARK: - HardcodedImageProvider
+
+/// An `AnimationImageProvider` that always returns a specific hardcoded image of a dog (hi Marley)
+struct HardcodedImageProvider: AnimationImageProvider {
+  func imageForAsset(asset _: ImageAsset) -> CGImage? {
+    #if os(iOS)
+    return UIImage(named: "Samples/Images/dog.png", in: .module, compatibleWith: nil)?.cgImage
+    #else
+    return nil
+    #endif
+  }
+}
```

Comment:
nit: add a `name` property to this provider so we can use it in other places?
```suggestion
struct HardcodedImageProvider: AnimationImageProvider {
  var name: String
  
  func imageForAsset(asset _: ImageAsset) -> CGImage? {
    #if os(iOS)
    return UIImage(named: name, in: .module, compatibleWith: nil)?.cgImage
    #else
    return nil
    #endif
  }
}
```

Reply:
good idea